### PR TITLE
Newsletter 13: Add a section about puffin profiler

### PR DESCRIPTION
**Pull request by emilk**
https://github.com/rust-gamedev/rust-gamedev.github.io/issues/232

### CODE REVIEW

File: content/posts/newsletter-013/index.md [hunk 1]
```diff
@@ -66,6 +66,30 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Library & Tooling Updates
 
+### 🐦 [Puffin Profiler]
+
+Pufin is a simple instrumentation profiler created by [Embark]
+where you opt-in to profile parts of you code.
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-013/index.md:72]:
```suggestion
where can you opt-in to profile parts of your code.
```

**Review by 17cupsofcoffee — APPROVED** [review]
Suggested a very minor tweak to the phrasing of the opening paragraph, but otherwise this looks good.

**Review by ozkriff — APPROVED** [review]

**Comment by ozkriff** [comment]
@emilk could you either resolve the conflicts or enable the "allow edits from maintainers." checkbox, please?

File: content/posts/newsletter-013/index.md [hunk 2]
```diff
@@ -66,6 +66,30 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Library & Tooling Updates
 
+### 🐦 [Puffin Profiler]
+
+Pufin is a simple instrumentation profiler created by [Embark]
+where you can opt-in to profile parts of your code.
+
+``` rust
```
Comment (ozkriff) [inline content/posts/newsletter-013/index.md:74]: (fmt nitpick) could you also remove this space here, please?

**Comment by emilk** [comment]
> @emilk could you either resolve the conflicts or enable the "allow edits from maintainers." checkbox, please?

Where is that checkbox?

**Comment by ozkriff** [comment]
Merged, thanks!

> where is that checkbox you speak of? I see it not.

Should be in the right panel:

image

https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork

**Comment by emilk** [comment]
It's not there for me.

**Pull request merged**